components/ProgressView/src/Theming/MDCProgressView+MaterialTheming.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialContainerScheme.h"
+#import "MaterialProgressView.h"
+
+/**
+ Applies an object conforming to MDCContainerScheming to theme a MDCProgressView.
```

Comment:
Nit: Doesn't this describe the method below and not the category generally?  I would expect it to say something like,

> This category is used to style MDCButtons instances to a specific Material style which can be found
> within the [Material Guidelines](https://material.io/design/components/buttons.html).